src/styles/blocks/stars.css

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star,
+.stars--1 :nth-child(n + 1),
+.stars--2 :nth-child(n + 2),
+.stars--3 :nth-child(n + 3),
+.stars--4 :nth-child(n + 4) {
+  background-image: url(../../images/star.svg);
```

Comment:
You can set this image by default to the `.stars__star` class and change it using nth-child rule